recipes/libgab/build.sh

```diff
@@ -10,8 +10,11 @@ mkdir -p ${PREFIX}/lib/libgab/gzstream/
 
 
 make  CXX="${CXX}"   all
-cp libgab*a ${PREFIX}/lib/libgab/
-cp     *\.h ${PREFIX}/include/libgab/
+cp     libgab*a               ${PREFIX}/lib/libgab/
+cp     PutProgramInHeader.o   ${PREFIX}/lib/libgab/ 2>/dev/null || :
+cp     BamFunctions.o         ${PREFIX}/lib/libgab/ 2>/dev/null || :
+cp     ReconsReferenceBAM.o   ${PREFIX}/lib/libgab/ 2>/dev/null || :
```

Comment:
1) These aren't getting copied into the package (must be an error in that's suppressed by redirecting to /dev/null).
2) If packages need functions in these then they should be in static or dynamic libraries, not object files.